spec/models/searchgov_domain_spec.rb

```diff
@@ -431,79 +404,79 @@
 
     context 'when there is no robots.txt' do
```

Comment:
Would it make sense to add a `context` for `#sitemap_urls` here for when `http` is used?

Reply:
No, because the `sitemap_urls` method will always return `https` URLs:
https://github.com/GSA/search-gov/pull/1190/files#diff-3b12b8e6b938e033ac47da6f7612a003044090621d665af314b50ca29cbcdc57R72